nautobot/extras/migrations/0023_auto_20220222_1755.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.1.14 on 2022-02-22 17:55
+
+from django.db import migrations, models
+import nautobot.core.celery
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("extras", "0022_objectchange_object_datav2"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="objectchange",
+            name="object_datav2",
+            field=models.JSONField(
+                blank=True, editable=False, encoder=nautobot.core.celery.NautobotKombuJSONEncoder, null=True
```

Comment:
Maybe add a comment?
```suggestion
                blank=True,
                editable=False,
                # The default DjangoJSONEncoder doesn't handle conversion of `set` objects to JSON
                encoder=nautobot.core.celery.NautobotKombuJSONEncoder,
                null=True,
```